src/auth_db/test.rs

```diff
@@ -26,25 +142,62 @@ fn get_bounces_invalid_address() {
 }
 
 #[test]
-fn deserialize_bounce_type() {
-    let bounce_type: BounceType = serde_json::from_value(json!(0)).expect("JSON error");
-    assert_eq!(bounce_type, BounceType::Soft);
-    let bounce_type: BounceType = serde_json::from_value(json!(1)).expect("JSON error");
-    assert_eq!(bounce_type, BounceType::Hard);
-    let bounce_type: BounceType = serde_json::from_value(json!(2)).expect("JSON error");
-    assert_eq!(bounce_type, BounceType::Soft);
-    let bounce_type: BounceType = serde_json::from_value(json!(3)).expect("JSON error");
-    assert_eq!(bounce_type, BounceType::Complaint);
-}
+fn create_bounce() {
+    let settings = Settings::new().expect("config error");
+    let db = DbClient::new(&settings);
 
-#[test]
-fn deserialize_invalid_bounce_type() {
-    match serde_json::from_value::<BounceType>(json!(4)) {
-        Ok(_) => assert!(false, "serde_json::from_value should have failed"),
-        Err(error) => assert_eq!(error.description(), "JSON error"),
-    }
-    match serde_json::from_value::<BounceType>(json!(-1)) {
-        Ok(_) => assert!(false, "serde_json::from_value should have failed"),
-        Err(error) => assert_eq!(error.description(), "JSON error"),
+    // The email addresses in this test are pseudo-namespaced so that we don't
+    // accidentally create bounce records that cause other tests to fail.
```

Comment:
Because tests run in parallel, or because we don't clean them up after the test completes?